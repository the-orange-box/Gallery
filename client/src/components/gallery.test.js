import React from 'react';
import { shallow } from 'enzyme';
import Gallery from './gallery.jsx';

describe('Gallery Component', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Gallery handleClick={()=>{}} imagelist={[]} debug/>);
  
    expect(component).toMatchSnapshot();
  });
});