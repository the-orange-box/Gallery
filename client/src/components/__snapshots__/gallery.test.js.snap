// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery Component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
